### ISSUE TITLE
security audit for UrlModifier

### ISSUE THREAD

**Issue by adam-ce** [AlpineMapsOrg/renderer#42]
malicious users could now send hacked urls. we should check for good measure. now it's not yet important, but in the future we might handle passwords etc.

**Comment by GeraldKimmersdorfer** [comment]
good thing were not working with database queries quite yet ;) and since we work with c++ we have to cast all strings to appropriate formats anyway. The only reasonable point of attack is the gl configuration but also this gets deserialized using qt functionaly. So the only thing that might happen is that the local instance crashes. 

I totally agree though that as soon as we jiggle around data in a server/client environment, we have to play close attention to security.

**Comment by adam-ce** [comment]
yes, i agree to all of it. I just put it as a reminder to think about potential attack vectors etc by myself. and maybe talk to a security expert that i know :)
